api/openapi-spec/swagger.json

```diff
@@ -10663,6 +10714,57 @@
      }
     ]
    },
+   "/apis/subresources.kubevirt.io/v1alpha3/namespaces/{namespace:[a-z0-9][a-z0-9\\-]*}/virtualmachines/{name:[a-z0-9][a-z0-9\\-]*}/memorydump": {
```

Comment:
Should we be adding this to v1alpha3 as well?

Reply:
this is a autogenerated file

Reply:
right, i thought you added it to that API version as well, but that is not the case